src/archiefvernietigingscomponent/templates/demo/index.html

```diff
@@ -0,0 +1,122 @@
+{% extends "demo/master.html" %}
+
+{% block content %}
+
+  <div class="px-3 py-3 pt-md-5 pb-md-4 mx-auto text-center">
+    <h1 class="display-4">{{ settings.PROJECT_NAME }}</h1>
+    <p class="lead">{{ settings.SITE_TITLE }}</p>
+  </div>
+
+  <div class="container text-center">
+
+    <ul class="nav nav-tabs justify-content-center" id="languageChoice" role="tablist">
+      <li class="nav-item ">
+        <a class="nav-link text-info" id="en-tab" data-toggle="tab" href="#en" role="tab" aria-controls="en" aria-selected="true">English</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link active text-info" id="nl-tab" data-toggle="tab" href="#nl" role="tab" aria-controls="nl" aria-selected="false">Nederlands</a>
+      </li>
+    </ul>
+    <div class="tab-content" id="description">
+      <div class="tab-pane fade" id="en" role="tabpanel" aria-labelledby="en-tab">
+
+        <!-- English -->
+        <br>
+        <p>
+          Record management is the practice of destroying data-records that have met their
+          end-of-life time. "Zaakgericht werken" prescribes the archiving terms for cases
+          that have been brought to an end.
+        </p>
+        <p>
+          The Archiefvernietigingscomponent (AVC) provides functionality for record
+          managers to schedule destruction of cases according to the archiving parameters.
+          It implements a multi-step, multi-role accordance process.
+        </p>
+
+        {% if settings.AVC_DEMO_MODE %}
+
+        <p>
+          The <strong>demo mode</strong> is enabled and allows you to use this application, as seen from different
+          roles. When you open the application, a tutorial is shown on the right side of the screen that explains all
+          steps. You can freely explore this demo and even see the results in the configured
+          <a class="text-info" href="https://opengem.nl/producten/open-zaak/">Open Zaak</a> instance. The demo data
+          will be reset multiple times per day.
+        </p>
+
+        <p>
+          Here are the roles available in the demo:
+          <ul>
+            <li>Functional administrator</li>
+            <li>Process owner</li>
+            <li>Archiver</li>
+            <li>Record manager</li>
+          </ul>
+        </p>
+
+        <p>
+          <a href="{% url 'admin:index' %}" class="btn btn-sm btn-info"><i class="fas fa-chalkboard-teacher"></i> Start tutorial</a>
+        </p>
```

Comment:
Yes! And now looking at it I am not sure that this button is really needed in Demo mode, because one can automatically log in from the side bar :thinking:

Reply:
I have added a button in non-Demo mode and removed it from the Demo mode. Although now there are an 'Administration' button and a 'Log in' button, both directing to the admin log in page.

Reply:
Let's redirect the "Zelfstudie" to the login role for recordmanager.
And yes, the administration button should be shown at all times in that page.

Reply:
After discussion: since the 'login' and 'administration' button would both redirect to the admin login page, having only the 'administration' button is enough